### DISCUSSION TITLE
Servers on github codespace

### DISCUSSION THREAD

**Discussion by dbtprods** [community/community#52747] [Codespaces]
### Select Topic Area

General

### Body

I have a node js server for a website and someone told me that if I use github codespace I could just keep it running forever for free is that true or is github codespace paid?

**Comment by osortega** [comment]
Hello @dbtprods!

That is not entirely true, GitHub Codespaces does offer a free tier for personal accounts that would allow you to use the product for up to 120 core hours, or 60 hours of run time on a 2 core codespace per month.

You can find more information about billing [here](https://docs.github.com/en/billing/managing-billing-for-github-codespaces/about-billing-for-github-codespaces).

Hope this helps!
